### PR TITLE
[KT] Skip all test cases with an error "LLVM ERROR: SLM size exceeds target limits"

### PR DESCRIPTION
**Pull request by SergeyKopienko**
In this PR skip all test cases with an error "LLVM ERROR: SLM size exceeds target limits"

### CODE REVIEW

File: test/kt/esimd_radix_sort.cpp [hunk 1]
```diff
@@ -37,6 +37,7 @@
 #include <iostream>
 #include <cmath>
 #include <limits>
+#include <string_view>
```
Comment (dmitriy-sobolev) [inline test/kt/esimd_radix_sort.cpp:40]: What is the purpose of this include?

File: include/oneapi/dpl/experimental/kt/esimd_radix_sort.h [hunk 2]
```diff
@@ -31,6 +31,15 @@ template <bool _IsAscending, std::uint8_t _RadixBits, typename _KernelParam, typ
 sycl::event
 radix_sort(sycl::queue __q, _Range&& __rng, _KernelParam __param)
 {
+    static_assert(_RadixBits == 7 || _RadixBits == 8);
```
Comment (dmitriy-sobolev) [inline include/oneapi/dpl/experimental/kt/esimd_radix_sort.h:34]: I would suggest leaving only 8 for now. 7 has not been verified with the majority of configurations.

File: include/oneapi/dpl/experimental/kt/esimd_radix_sort.h [hunk 3]
```diff
@@ -31,6 +31,15 @@ template <bool _IsAscending, std::uint8_t _RadixBits, typename _KernelParam, typ
...
+
+    static_assert(param.data_per_workitem == 32 || param.data_per_workitem == 64 || param.data_per_workitem == 96 ||
+                  param.data_per_workitem == 128 || param.data_per_workitem == 160 || param.data_per_workitem == 192 ||
+                  param.data_per_workitem == 224 || param.data_per_workitem == 256 || param.data_per_workitem == 288 ||
+                  param.data_per_workitem == 320 || param.data_per_workitem == 352 || param.data_per_workitem == 384 ||
+                  param.data_per_workitem == 416 || param.data_per_workitem == 448 || param.data_per_workitem == 480 ||
+                  param.data_per_workitem == 512);
```
Comment (dmitriy-sobolev) [inline include/oneapi/dpl/experimental/kt/esimd_radix_sort.h]: Can you use `%` for simplicity?

File: include/oneapi/dpl/experimental/kt/esimd_radix_sort.h (hunk 2 above)
Comment (SergeyKopienko) [reply include/oneapi/dpl/experimental/kt/esimd_radix_sort.h]: Fixed

File: test/kt/esimd_radix_sort.cpp (hunk 1 above)
Comment (SergeyKopienko) [reply test/kt/esimd_radix_sort.cpp]: Removed

File: include/oneapi/dpl/experimental/kt/esimd_radix_sort.h (hunk 3 above)
Comment (SergeyKopienko) [reply include/oneapi/dpl/experimental/kt/esimd_radix_sort.h]: As for me better to have full list of available values without any calculations.

File: test/kt/esimd_radix_sort.cpp [hunk 4]
```diff
@@ -386,38 +388,53 @@ void test_general_cases(std::size_t size, KernelParam param)
 }
 #endif // TEST_DPCPP_BACKEND_PRESENT
 
+template <typename T, typename KernelParam>
+bool
+can_run_test(KernelParam param)
+{
+    // uint64_t, int64_t, double - skip tests with error: LLVM ERROR: SLM size exceeds target limits
+    if (sizeof(T) == sizeof(::std::uint64_t) && param.data_per_workitem >= 288 && param.workgroup_size == 64)
```
Comment (dmitriy-sobolev) [inline test/kt/esimd_radix_sort.cpp:396]: Could you investigate how much memory is used by the implementations and compare it against available memory, so the check is more complete? 

`info::device::local_mem_size` device descriptor shows available memory. Our experiments have shown that compiler can implicitly use some SLM, so we can utilize only a part of the device SLM. 
Example: 
https://github.com/oneapi-src/oneDPL/blob/8528820d414921ce04169157e212d5bc6421ba1e/include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h#L912

An example is capped to only a half of SLM, I consider it is too strict for ESIMD kernels basing on testing experience. Could you find the most appropriate coefficient?

**Review by dmitriy-sobolev — APPROVED** [review]

**Pull request merged**